content/en/apps/guides/messaging/rapidpro.md

```diff
@@ -0,0 +1,114 @@
+---
+title: RapidPro SMS Gateway
+linkTitle: "RapidPro"
+weight:
+description: >
+    Integration for sending and receiving SMS
+relatedContent: >
+    apps/reference/app-settings/sms
+    apps/guides/forms/app-form-sms
+    apps/guides/messaging/sms-states
+    apps/guides/messaging/message-loops
+    apps/guides/messaging/shortcodes
+---
+
+
+As of v3.11.0, SMS messages can be sent and received using [RapidPro](https://community.rapidpro.io/).
+
+## RapidPro configuration
+
```

Comment:
I think the first question we'll get is "how do I access rapidpro?". I think it'd be worth adding something about step 1 being contact Medic to host, or use TextIt, or host your own.

Reply:
There's already a section about RapidPro here: https://docs.communityhealthtoolkit.org/apps/features/integrations/rapidpro/

Would it suite to link to that and reduce duplicate content?

Reply:
Yes, linking is perfect, thanks.